content/blog/2023-06-07-oreoscan

```diff
@@ -0,0 +1,61 @@
+author: Elena 
+title: "Ecosystem Spotlight: Oreoscan"
+description: Learn more about Oreoscan, an incredible community member building on the Iron Fish network
+image: public/images/ecosystem/oreoscan.png
+tags: [community-highlight]
```

Comment:
This is missing the `---` before and after the header section. Look at another file like https://github.com/iron-fish/website/blob/master/content/blog/2023-07-05-rfp-governance.mdx for an example